packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config-editor.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+	<div class="value-editor">
+		<section>
+			<div>
+				<h4>Initials</h4>
+				<Accordion v-if="stratifiedModelType && !_.isEmpty(initials)" :multiple="true">
+					<AccordionTab v-for="[init, vals] in initials" :key="init">
+						<template #header>
+							{{ init }}
+						</template>
+						<ul>
+							<li v-for="val in vals" :key="val">
+								<Button
+									:label="val"
+									class="asset-button"
+									plain
+									text
+									size="small"
+									@click="getClickedField('initial', val)"
+								/>
+							</li>
+						</ul>
+					</AccordionTab>
+				</Accordion>
+				<ul v-else-if="!stratifiedModelType && !_.isEmpty(initials)">
+					<li v-for="[init, vals] in initials" :key="init">
+						<Button
+							:label="init"
+							class="asset-button"
+							plain
+							text
+							size="large"
+							@click="getClickedField('initial', vals[0])"
+						/>
+					</li>
+				</ul>
+			</div>
+			<div>
+				<h4>Parameters</h4>
+				<Accordion v-if="stratifiedModelType && !_.isEmpty(parameters)" :multiple="true">
+					<AccordionTab v-for="[param, vals] in parameters" :key="param">
+						<template #header>
+							{{ param }}
+						</template>
+						<ul>
+							<li v-for="val in vals" :key="val">
+								<Button
+									:label="val"
+									class="asset-button"
+									plain
+									text
+									size="small"
+									@click="getClickedField('parameter', val)"
+								/>
+							</li>
+						</ul>
+					</AccordionTab>
+				</Accordion>
+				<ul v-else-if="!stratifiedModelType && !_.isEmpty(parameters)">
+					<li v-for="[param, vals] in parameters" :key="param">
+						<Button
+							:label="param"
+							class="asset-button"
+							plain
+							text
+							size="large"
+							@click="getClickedField('parameter', vals[0])"
+						/>
+					</li>
+				</ul>
+			</div>
+		</section>
```

Comment:
Okay for now, but this `<section>` is a bit wonky to use if when you have a long initials/parameters list, you have to constantly scroll back and forth. Let's flag this as a FIXME.

Reply:
fixme added